packages/frontend/scss/message/_message-attachment.scss

```diff
@@ -31,6 +31,9 @@
     object-position: center;
     max-width: 450px;
     min-width: 200px;
+    &.sticker {
+      min-width: auto;
+    }
```

Comment:
I think this can also be moved to here, making the `message.viewType === 'Sticker' ? 'sticker' : null` unnecessary.

https://github.com/deltachat/deltachat-desktop/blob/ff6b346ed393c28b3d73a562bf7e3ba9807ea72a/packages/frontend/scss/message/_message.scss#L299-L309

Reply:
Turned out there is still an edge case: on screens smaller 900px the message width is smaller than the image min-width so we need to set the width to "auto" but not for stickers, since they are limited to 200px height